dashboard/test/controllers/lti_v1_controller_test.rb

```diff
@@ -49,45 +49,50 @@ def create_valid_jwt_raise_error
     create_jwt_and_stub(payload, true)
   end
 
-  test 'given no params, return unauthorized' do
+  def read_cache(key)
+    json_value = CDO.shared_cache.read(key)
+    JSON.parse(json_value).symbolize_keys
+  end
```

Comment:
do we want to read from the cache in the tests or just stub out the cache (check that write was called)?

Reply:
Yeah... probably stub it.